projects/plugins/jetpack/extensions/blocks/ai-image/edit.js

```diff
@@ -14,6 +14,25 @@ import { useSelect, useDispatch } from '@wordpress/data';
 import { useState } from '@wordpress/element';
 import { __ } from '@wordpress/i18n';
 
+function getRandomItem( arr ) {
+	// get random index value
+	const randomIndex = Math.floor( Math.random() * arr.length );
+	return arr[ randomIndex ];
+}
+
+const PLACEHOLDER = getRandomItem( [
```

Comment:
By having randomness here and not in Render, when you insert the block twice into the same post, you will have the same prompts.
So it looks like there is only one prompt.

Reply:
Done.